app/views/provider/admin/dashboards/_services_filter.html.slim

```diff
@@ -6,11 +6,14 @@
         var allApis = #{json @services}
```

Comment:
Does this really expose all the service attributes? https://github.com/3scale/porta/blob/b0940c01de323748f9b98db6a066f41eb9bac086/db/schema.rb#L1104-L1145
Shouldn't it be limited to some meaningful scope?